packages/eth2.0-constants/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@chainsafe/eth2.0-constants",
```

Comment:
I would maybe expose this as eth2.0-utils I don't see a reason for this to be standalone package

Reply:
Biggest consideration is this package can't rely on eth2.0-config / eth2.0-ssz-types.
We're pretty good on that with our src/util directory (other than src/util/deposits, which can be moved)

Reply:
yeah, current candidates are hash, merkle tree and math utils. I don't think utils should depend on those packages. If something is depending it ain't suppose to go to utils :D

Reply:
removed constants package, brought back into lodestar